data/bangs.json

```diff
@@ -107355,5 +107355,13 @@
     "u": "https://libro.fm/search?utf8=%E2%9C%93&q={{{s}}}",
     "c": "Shopping",
     "sc": "Online"
+  },
```

Comment:
Can you add this to somewhere in the middle of the repo instead of the end? Helps prevent merge conflicts